pom.xml

```diff
@@ -152,12 +152,12 @@
         <dependency>
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
-            <scope>test</scope>
+            <optional>true</optional>
```

Comment:
@bdragan I recall we should try to avoid usage of other frameworks in Takes 
it possible only in tests
@yegor256 please confirm